### DISCUSSION TITLE
New Version 6   Problem with Simbrief

### DISCUSSION THREAD

**Discussion by DieterKuep** [flybywiresim/aircraft#4458] [Help]
Hi there,
after updating to version 6.0 i get freezing for the sim if i try to import the simbrief flightplan.
by importing itself everythink works, but if i try to chanage some settings in the fmc suddenly msfs2020 1.15.7.0
stopps.
Any solution?
Thanks
Dieter

**Comment by CptBayer** [comment]
Hi,

with the experimental version I had no problems .... Try this ... 

Greets Klaus

**Reply by Paytheon** [reply]
I did try the Version 6 version and verify for certain there's a freezing issue. The experimental version doesn't have the freezing issue but most certainly has an issue with the flight plan go haywire when going into the approach phase of the flight plan. I guess it's a pick your poison decision to make. That being said, I still enjoy this aircraft and the work the developers are putting into making this plane work.

**Reply by DieterKuep** [reply]
OK thanks for reply
i tried the experimental version but i get the same fault.
After importing the simbrief plan i tried to include the sid and stars.
with the click on the departure airport (to get the menue) the sim freezes.
there is no way to going on withe msfs2020 - sorry
